infrastructure/instances.tf

```diff
@@ -285,17 +265,15 @@ output "nomad_client_ip" {
   value = "${aws_instance.nomad_client_1.public_ip}"
 }
 
-variable "database_password" {}
-
 resource "aws_db_instance" "postgres_db" {
   identifier = "data-refinery-${var.user}-${var.stage}"
   allocated_storage = 100
   storage_type = "gp2"
   engine = "postgres"
-  engine_version = "9.5.4"
+  engine_version = "9.5.10"
```

Comment:
I made this change because one `terraform apply` said that it was going to change the version from `9.5.10 -> 9.5.4` so I think RDS upgraded postgres for us, so we might as well explicitly get on the latest version anyway.